### DISCUSSION TITLE
How to access or perform a git commit in a deprecated codespace?

### DISCUSSION THREAD

**Discussion by AWATV** [community/community#52340] [Codespaces] [answered]
### Select Topic Area

Question

### Body

And I was writing my code, and I didn't manage to commit until the time ran out. Is there a way to get the code or commit the latest changes?

**Comment by osortega** [comment, marked as answer]
Hi @AWATV 👋 !
What do you mean by deprecated codespace?
You can always export the changes of a codespace to a branch, you just need to go to https://github.com/codespaces, click on the three dots on the right side of the codespace name and select `Export changes to a branch`

<img width="270" alt="Screenshot 2023-04-10 at 11 07 57 AM" src="https://user-images.githubusercontent.com/48293249/230964228-84c034ae-289c-4cf2-a5b2-ecd083ecb942.png">

Hope this helps!

**Reply by AWATV** [reply]
Thx for help!
